x/ibc-account/keeper/grpc_query.go

```diff
@@ -52,8 +52,7 @@ func (k Keeper) IBCAccountFromData(ctx context.Context, req *types.QueryIBCAccou
 	}
 
 	sdkCtx := sdk.UnwrapSDKContext(ctx)
-	identifier := types.GetIdentifier(req.Port, req.Channel)
-	address := k.GenerateAddress(identifier, []byte(req.Data))
+	address := k.GenerateAddress("test")
```

Comment:
I'm going to update the grpc_query stuff today, and probably add a new cli command for querying for interchain accounts.